### PR TITLE
bpo-12317: Update the documentation for inspect.getabsfile()

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue12317

### CODE REVIEW

**Comment by corona10** [comment]
@terryjreedy Please take a look!

File: Doc/library/inspect.rst
Comment (eamanu) [inline Doc/library/inspect.rst:248]: IMO this phrase could be remove because this sound like an "idea" and not like something concrete I do not know if I'm clear)

Comment (corona10) [reply Doc/library/inspect.rst:248]: @eamanu This is part of getabsfile docstring.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
If we document this function, the docstring and the derived doc entry should be modified.  This should be on hold until core devs agree on what to do.  See the issue.

**Pull request closed without merging**
